### DISCUSSION TITLE
Feature request: Open/edit multiple files at once

### DISCUSSION THREAD

**Discussion by macsunmood** [KDE/ghostwriter#886] [Ideas]
Hi! Does the requested feature go against some basic ghostwritter philosophy? Or not?
This could be implemented via tabs or selectable items in the left panel.

**Comment by Aryaman-chandra** [comment]
I too wanted to know this.

**Comment by MidoriMushrooms** [comment]
Older versions of Ghostwriter had this functionality. It seems to have been removed for some reason, along with the ability to exit without saving your edits, which is functionality I also miss having.

**Comment by macsunmood** [comment]
Any updates on this ?

**Comment by jfmdev** [comment]
I switched to Ghostwriter after Typora became a paid app, and this is the only feature that I'm missing: the ability to open a "project folder" from where to open the files that I need to edit.

**Reply by zylstra** [reply]
What about MarkText, https://github.com/marktext/marktext?

**Reply by jfmdev** [reply]
I tried MarkText but:

1. It don't has a "Split mode" view (where on the left you see plain text, and on the right you see the rendered text).
2. The application has the habitude of _formatting_ the text without asking permission (e.g. if you have three consecutive empty lines, it will automatically replace it with a single empty line after opening the file).
3. The project seems to be abandoned (last release was 3 years ago).
